recipe/meta.yaml

```
@@ -152,6 +154,88 @@ outputs:
       license: MIT
       license_file: LICENSE.TXT
 
+  - name: libgdal-arrow-parquet
+    script: build_arrow_parquet.sh  # [unix]
+    script: build_arrow_parquet.bat  # [win]
+    requirements:
+      build:
+        - cmake
+        # ranlib used at install time
+        - {{ compiler('c') }}
+        # libstdc++ is needed in requirements/run
+        - {{ compiler('cxx') }}
+        - pkg-config  # [not win]
+        - make  # [unix]
+        - ninja  # [win]
+        - sysroot_linux-64 2.17  # [linux64]
+      host:
+        - {{ pin_subpackage('libgdal', exact=True) }}
+        - arrow-cpp >=10.0
```

Having a lower bound is fine (but should be coupled with the skip, as mentioned above), but it's better to _also_ leave an unpinned line so that future versions will be built correctly against a single arrow version.

Also, if you're fine with only building for arrow >=10, please use the new name for the library, `arrow-cpp` is just there for compatibility and will eventually be removed.

```suggestion
        - libarrow
        - libarrow >=10.0
```